### PR TITLE
Port GHS sprites

### PR DESCRIPTION
**Pull request by VictorJob**
## About the PR

This PR ports the GHS sprites from CM13. Some sprites underwent small modifications and the suittorage sprite was made by me.

This PR also corrects the smartgun's "Icon" (previously written with the first letter capitalized) to "icon"
## Media

image

image

image

image

image

## Requirements

- [x] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [x] I have added media to this PR or it does not require an ingame showcase.

### CODE REVIEW

File: Resources/Textures/_RMC14/Objects/Weapons/Guns/SmartGuns/GSH.rsi/meta.json [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    },
+    {
+      "name": "bolt-open"
+    },
+    {
+      "name": "equipped-BACKPACK",
+      "directions": 4
+    },
+    {
+      "name": "Icon"
```
Comment (DrSmugleaf) [inline Resources/Textures/_RMC14/Objects/Weapons/Guns/SmartGuns/GSH.rsi/meta.json:36]: This needs to be lowercase, same for the corresponding filename

File: Resources/Textures/_RMC14/Objects/Weapons/Guns/SmartGuns/GSH.rsi/meta.json [hunk 2]
```diff
@@ -0,0 +1,47 @@
+﻿{
+  "version": 1,
+  "license": "CC-BY-SA-4.0",
+  "copyright": "Taken from cmss13 at https://github.com/cmss13-devs/cmss13/commit/3163b00, suiststorage sprites made by VictorJob",
```
Comment (DrSmugleaf) [inline Resources/Textures/_RMC14/Objects/Weapons/Guns/SmartGuns/GSH.rsi/meta.json:4]:
```suggestion
  "copyright": "Taken from cmss13 at https://github.com/cmss13-devs/cmss13/commit/3163b00, suitstorage sprites made by VictorJob",
```

**Review by DrSmugleaf — CHANGES_REQUESTED** [review]
Thanks for the PR!

File: Resources/Textures/_RMC14/Objects/Weapons/Guns/SmartGuns/GSH.rsi/meta.json (hunk 1 above)
Comment (VictorJob) [reply Resources/Textures/_RMC14/Objects/Weapons/Guns/SmartGuns/GSH.rsi/meta.json]: fixed, also fixed the smart gun that had an "Icon" in both the meta and the sprite name

File: Resources/Textures/_RMC14/Objects/Weapons/Guns/SmartGuns/GSH.rsi/meta.json [hunk 3]
```diff
@@ -0,0 +1,47 @@
+﻿{
+  "version": 1,
+  "license": "CC-BY-SA-4.0",
+  "copyright": "Taken from cmss13 at https://github.com/cmss13-devs/cmss13/commit/3163b00, suitstorage sprites made by VictorJob",
```
Comment (DrSmugleaf) [inline Resources/Textures/_RMC14/Objects/Weapons/Guns/SmartGuns/GSH.rsi/meta.json:4]: This needs to include the .dmi that the states were taken from, RSIEdit will do this automatically for you if you copy paste dmis into it
Example:
`Taken from cmss13 at https://github.com/cmss13-devs/cmss13/blob/7cb618c69b75873f3ce893022fe08d1233b3152d/icons/obj/items/weapons/guns/attachments/rail.dmi`

**Review by DrSmugleaf — CHANGES_REQUESTED** [review]

**Comment by VictorJob** [comment]
Hello. To make this clear, I deleted the repository so I could contribute to another community (their current codebase is Goobstation, but for some reason I couldn't create a fork there while having one here). If you want me to make any more sprites, you can call me on discord,
I just don't know if I'll be able to make PRs here from now on without having to delete and create forks.

**Pull request closed without merging**